lib/test.js

```diff
@@ -230,33 +251,23 @@ Test.prototype._publicApi = function () {
 	api.plan = this.plan.bind(this);
 
 	function skipFn() {
-		self._assert();
+		self._assert(Promise.resolve());
```

Comment:
Agreed. I wrote this before I saw how you optimized the `onAssertion` callback. You can disregard.